src/main.rs

```diff
@@ -216,7 +240,21 @@ async fn main() {
                 None => panic!("title must be provided"),
             };
             let ledger = false;
-            let message = create_message(password, href, title, ledger).await;
+            let message = create_message(password, href, title, ledger, None).await;
```

Comment:
```suggestion
            let message = create_message(password, href, title, ledger, args.address_index).await;
```

@chrsmaral do I understand this right that for submit-ledger to allow the WalletIndex argument, we have to pass address_index to create_message here, right?

Reply:
`Submit` and `Vote` passes 'None' as 5th argument when calling `create_message` as a placeholder (to not be wrongfully fixed, since it can be marked as an error by LSP's) and I don't like to hardcode a random value or empty variable. 

Only if the ledger flag in the function `create_message' is true, `address_index` is actually used. That won't be the case, so it doesn't matter. Only `SubmitLedger` and `VoteLedger` need that arg, due to the flag being set to true.

We could create a separate function for ledger wallets, but for now it should be good enough.
We can, of course, do it always at a later point. Especially when it gets more complex.

Reply:
Ah ok makes sense. Actually my bad. I thought this was editing the SubmitLedger function.